arm_num: 30
arm_unit_length: 0.013
vendor_num: 2
arm_unit_radius: 0.02
linear_vel: 0.015
max_unit_angle_degree: 20
base_frame_id: right_link

# 9 line 
# 8 circle
# 11 hyotan
target_pose_shape_num: 9